app/service/rest.py

```diff
@@ -213,14 +213,15 @@ def get_all_notifications_for_service(service_id):
     page = data['page'] if 'page' in data else 1
     page_size = data['page_size'] if 'page_size' in data else current_app.config.get('PAGE_SIZE')
     limit_days = data.get('limit_days')
+    include_jobs = False if ('include_jobs' in data and data['include_jobs'] is False) else True
```

Comment:
It’s actually `include_jobs = data.get('include_jobs', True)` but yeah, cleaner than the one line `if`.